exporters/otlp/protocoldriver.go

```diff
@@ -49,3 +50,83 @@ type ProtocolDriver interface {
 	// take this into account by doing proper locking.
 	ExportTraces(ctx context.Context, ss []*tracesdk.SpanSnapshot) error
 }
+
+type SplitConfig struct {
```

Comment:
nit: This is great. Please add a comment on this type. :-)

Reply:
Oops, fixed. Thanks.